pandas/core/arrays/string_arrow.py

```diff
@@ -85,7 +85,8 @@ class ArrowStringArray(ArrowExtensionArray, BaseStringArray, ObjectStringArrayMi
 
     See Also
     --------
-    array
+    `array
+    <https://pandas.pydata.org/docs/dev/reference/api/pandas.arrays.ArrowStringArray.html>`__
```

Comment:
Only one underscore? Not sure but doc build is failing

Reply:
There are two underscores, this was on the sphinx site to have two underscores